README.md

```diff
@@ -314,6 +286,50 @@ If we want to run a command on all search results, we can pipe the output to `xa
 Here, the `-0` option tells *fd* to separate search results by the NULL character (instead of
 newlines). In the same way, the `-0` option of `xargs` tells it to read the input in this way.
 
+### Parallel command execution
+If the `-x`/`--exec` option is specified alongside a command template, a job pool will be created
+for executing commands in parallel for each discovered path as the input. The number of threads 
+used for command execution can be set with the `--threads`/`-j` option.
+
+*fd* takes the command template as a series of arguments rather than as a string. If you want to 
+add additional options after the command template, you can terminate it with a `\;`. This is 
+useful when you want to repeat a command with new options. For example, to remove write and 
+execute permissions from all directories, run:
```

Comment:
It would be great if we could mention this whole `\;` business at the end of the section (before the threads option). I use `fd -x` and `fd -X` all the time and I never use `\;`. I'd rather teach the most common use cases first.

Reply:
Sure, that makes sense. As long as it is there is some clarification anywhere in the section, I think it will be helpful.